packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,210 @@
+.rt-DataListRoot {
+  display: grid;
+  grid-template-columns: auto 1fr;
+}
+
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
```

Comment:
- Same comment about including the CSS property conditionally based on the prop value
- Don't think we need `box-sizing: border-box` as there's no padding, also not sure about `position: relative`